John --> master: Added Luma NVR and Wirepath Camera support

jhallard opened the pull request:
I'm looking it over one more time, I believe everything listed here is correct.

jhallard committed:
Fixes #77: Add support for Luma NVR and Wirepath Cameras

Note -- the OUIs are missing, couldn't find that information through a
simple Google search. When we get the missing OUIs I can add that in
then the entries will be complete.

jhallard committed:
Fixed #77: Added in the OUI sections for the Luma NVR/Wirepath cam

That should be everything needed for these two new entries

jcmaslan commented on cameras/paths.csv:
```diff
@@ -36,3 +36,5 @@ Grandstream,camera,GXV3674_[F]HD_VF,(00:0B:82),rtsp://{{username}}:{{password}}@
 Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}}.sdp,H.264,554,live;live2;liveany,,Stream 1;Stream 2;Stream 3,,live2,live,,,,http://download.vivotek.com/downloadfile/downloads/usersmanuals/cc8130manual_en.pdf
 Dahua,camera,HFW81200,(4C:11:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}},H.264,554,,,,,,,admin,admin,,http://www1.dahuasecurity.com/products/ipc-hfw81200e-z-1159.html
 WatchNet,nvr,WatchNet DVR,(52:54:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16,High;Low,Channel {{channel}},1,0,admin,1234,,http://watchnetinc.com/public/pdfs/VMS%20Lite%20Users%20Manual_Ver1.0.pdf
+Luma,nvr,Lum-500/501 Series,(00:0F:0D),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,main stream;sub stream,Channel {{channel}},01,02,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/Luma-500%20Technical%20Referen ...
```
see my issue comment that the actual port is the concatenation of a base port and the channel.

jcmaslan commented on cameras/paths.csv:
```diff
@@ -36,3 +36,5 @@ Grandstream,camera,GXV3674_[F]HD_VF,(00:0B:82),rtsp://{{username}}:{{password}}@
 Dahua,camera,HFW81200,(4C:11:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}},H.264,554,,,,,,,admin,admin,,http://www1.dahuasecurity.com/products/ipc-hfw81200e-z-1159.html
 WatchNet,nvr,WatchNet DVR,(52:54:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16,High;Low,Channel {{channel}},1,0,admin,1234,,http://watchnetinc.com/public/pdfs/VMS%20Lite%20Users%20Manual_Ver1.0.pdf
+Luma,nvr,Lum-500/501 Series,(00:0F:0D),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,main stream;sub stream,Channel {{channel}},01,02,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/Luma-500%20Technical%20Referen ...
+Wirepath,camera,Wirepath IP Camera,(00:0F:0D|D4:6A:91),http://{{username}}:{{password}}@{{ip_Address}}/GetData.cgi?CH={{channel}}&Codec=h264,H.264,554,,1,,Channel {{channel}},,,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ProtocolsAndDrivers/Streaming_Protocols_20160613.pdf
```
Let's not repeat the brand/maker in the model field, so just say "IP Camera" instead.
Same goes for the Removal of WatchNet from the model in 2 rows above. thanks.

jhallard committed:
Changes for #77: concat port and channel for Luma, etc.

Also changed a typo ip_Address -> ip_address, and removed the repeating
of maker names from the NVR/Camera descriptions.

jhallard commented:
see 79b515a for changes

jcmaslan commented:
LGTM

jhallard commented:
Deploy is failing because the URL is of the form http://* instead of rtsp://, Edson had hardcoded a check into the Cams repo to make sure that the URL starts with rtsp. Should I change the Cams repo to remove this requirement or should we find a RTSP URL for the Wirepath camera?

jhallard commented:
Filed an issue with the Cams repo: https://github.com/CamioCam/Cameras/issues/30. When that's solved I'll come back here, deploy, then close this.

jhallard commented:
Fixed the Cams issue with (https://github.com/CamioCam/Cameras/pull/31). Getting it deployed then I'll deploy these changes.

jhallard commented:
Luma and Wirepath values deployed

```
100 22638  100 22638    0     0  98550      0 --:--:-- --:--:-- --:--:-- 98855
[
  {
    "rtsp_server": "rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}{{channel}}",
    "model": "Lum-500/501 Series",
    "default_values": {
      "username": {
        "is_multiselect": false,
        "options": [
          {
            "value": "admin"
          }
        ]
      },
      "password": {
        "is_multiselect": false,
        "options": [
          {
            "value": "admin"
          }
        ]
      },
      "port": {
        "is_multiselect": false,
        "options": [
          {
            "value": "554"
          }
        ]
      },
      "stream": {
        "is_multiselect": false,
        "options": [
          {
            "name": "main stream",
            "value": "01"
          },
          {
            "name": "sub stream",
            "value": "02"
          }
        ]
      },
      "channel": {
        "is_multiselect": true,
        "options": [
          {
            "name": "Channel 1",
            "value": "1"
          },
          {
            "name": "Channel 2",
            "value": "2"
          },
          {
            "name": "Channel 3",
            "value": "3"
          },
          {
            "name": "Channel 4",
            "value": "4"
          },
          {
            "name": "Channel 5",
            "value": "5"
          },
          {
            "name": "Channel 6",
            "value": "6"
          },
          {
            "name": "Channel 7",
            "value": "7"
          },
          {
            "name": "Channel 8",
            "value": "8"
          },
          {
            "name": "Channel 9",
            "value": "9"
          },
          {
            "name": "Channel 10",
            "value": "10"
          },
          {
            "name": "Channel 11",
            "value": "11"
          },
          {
            "name": "Channel 12",
            "value": "12"
          },
          {
            "name": "Channel 13",
            "value": "13"
          },
          {
            "name": "Channel 14",
            "value": "14"
          },
          {
            "name": "Channel 15",
            "value": "15"
          },
          {
            "name": "Channel 16",
            "value": "16"
          }
        ]
      }
    },
    "rtsp_path": "/Streaming/channels/{{channel}}{{stream}}",
    "maker": "Luma"
  }
]

john@Carters-MacBook-Pro-2: ~/Cameras   (john)
$ curl camio-cameras.appspot.com/paths | jq '.Wirepath'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 22638  100 22638    0     0   119k      0 --:--:-- --:--:-- --:--:--  120k
[
  {
    "rtsp_server": "http://{{username}}:{{password}}@{{ip_address}}",
    "model": "IP Camera",
    "default_values": {
      "username": {
        "is_multiselect": false,
        "options": [
          {
            "value": "admin"
          }
        ]
      },
      "password": {
        "is_multiselect": false,
        "options": [
          {
            "value": "admin"
          }
        ]
      },
      "port": {
        "is_multiselect": false,
        "options": [
          {
            "value": "554"
          }
        ]
      },
      "channel": {
        "is_multiselect": false,
        "options": [
          {
            "name": "Channel 1",
            "value": "1"
          }
        ]
      }
    },
    "rtsp_path": "/GetData.cgi?CH={{channel}}&Codec=h264",
    "maker": "Wirepath"
  }
]
```

jhallard commented:
New values are in the repo and have been deployed to testing+production, closing now.
